api/src/main/resources/api/service_openapi.yaml

```diff
@@ -525,6 +525,9 @@ components:
         relatedEntity:
           type: string
           description: The name of the related entity.
+        filterable:
+          type: boolean
+          description: Whether this relationship should be used when creating relationships.
```

Comment:
nit: is this comment correct? seems circular

Reply:
thanks, fixed up